### ISSUE TITLE
No icon images are showing

### ISSUE THREAD

**Issue by spellfork** [pixolith/fabricjs-customise-controls-extension#10]
I have problem showing the icon images. I have followed the examples to point and I can find the image resources in my debugger. Any suggestions?

These are my global settings, I get the right corners and the corners do the right action, but I do not see any images (but I can see that the images are loaded by the browser):

```
fabric.Canvas.prototype.customiseControls( {
    tl: {
        action: 'rotate',
        cursor: 'pointer'
    },
    br: {
        action: 'remove',
        cursor: 'pointer'
    }
} );

// basic settings
fabric.Object.prototype.customiseCornerIcons({
    settings: {
        borderColor: '#f0f',
        cornerSize: 40,
        cornerShape: 'circle',
        cornerBackgroundColor: '#f0f',
        cornerPadding: 10
    },
    tl: {
        icon: 'img/symbolRotate.svg'
    },
    br: {
        icon: 'img/symbolDelete.svg'
    }
});
```

**Comment by MDSLKTR** [comment]
Hello there and sorry for the late response.
First of thank you for you report. Can you by any chance attach one of these svgs here? It might be related to the way they are built or in general svg related. Otherwise any demo environment would be fine too. Whatever you prefer really. I will then be able to have closer look at what might be breaking the extension.

Other useful info would be what fabric version you are using since the new 1.6.0 release is freshly out and what Browser you are experiencing this in.

Simon

**Comment by spellfork** [comment]
Hi, and thanks for replying

I should have provided more info in my first post. I am using the 1.5.0 release of fabric and while dubugging I even tried to switch the file I had for the fabric.js provided in the examples folder, but it didn't fix my issue.

While trying to debug I was also using the .svg files in the example folder and I've had the same issue with those as with my own files. I have also tried this with .png files and still didn't get the image icon to show. I have tried it with or without a background color for the corner (in case there was some kind of layering/index problem).

The example code works for me, as does the web demo. I will try and set up some kind of demo tomorrow, the issue is most likely caused by something in my code. Worth mentioning is that I have no other extensions to fabric.js other than this one but I run my application on a bootstrap/jQuery built site.

Also I only use svg files in my app and load them with fabric.loadSVGfromURL() and in the callback I group the elements and do some scaling etc.

`var oImg = fabric.util.groupSVGElements(objects, newOptions);
oImg.scale(1);
canvas.add(oImg).centerObject(oImg).setActiveObject(oImg);
oImg.setCoords();`
